modules/Display.js

```diff
@@ -1,6 +1,4 @@
-Processes.listOfDevices['display'] = {
-    name: "Display",
```

Comment:
Just wondering why these were removed in all of the devices? I have an idea as to why, but want to hear your explanation